templates/example_cell_clustering.ipynb

```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
```

Comment:
<p>I know you mentioned when we talked that you wanted to maybe change the default prefix, but I think it's fine to keep it as the date. Maybe just add a line that says "If you didn't send a default in the pixel clustering notebook, the default was the timestamp. Look for that in the run folder and change the variable here" or something like that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>